guides/source/autoloading_and_reloading_constants.md

```diff
@@ -151,18 +151,17 @@ as you can see, the class object stored in the `User` constant is different afte
 
 It is very important to understand that Ruby does not have a way to truly reload classes and modules in memory, and have that reflected everywhere they are already used. Technically, "unloading" the `User` class means removing the `User` constant via `Object.send(:remove_const, "User")`.
 
-Therefore, if you store a reloadable class or module object in a place that is not reloaded, that value is going to become stale.
+Therefore, if you store a reloadable class or module object in a place that is not reloaded, that value will become stale.
```

Comment:
Curious about this edit, does it sound better to a native speaker?

Reply:
I was aiming for more of a factual/past tense word that drives the point to the reader, if you change the content of that class it's stale. Rather than saying if you change it, it's going to become stale.

I'm happy to revert this tbh, but it felt more clearer to me. Let me know how you wanna proceed?